src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -77,16 +82,18 @@ def create_webapp(cmd, resource_group_name, name, plan, runtime=None, startup_fi
             if not match:
                 raise CLIError("Linux Runtime '{}' is not supported."
                                "Please invoke 'list-runtimes' to cross check".format(runtime))
-
         elif deployment_container_image_name:
             site_config.linux_fx_version = _format_linux_fx_version(deployment_container_image_name)
             site_config.app_settings.append(NameValuePair("WEBSITES_ENABLE_APP_SERVICE_STORAGE", "false"))
-        else:  # must specify runtime
```

Comment:
why you remove this? Now we won't report anything if none of the arguments are supplied

Reply:
validate_linux_create_options() will catch this right?

Reply:
okay